blog/_posts/2017-09-19-how-to-survive-your-first-tech-conference.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
could you move this into `_drafts` (rather than `_posts`) for now? We'll be publishing it later this autumn, probably in November, if that's ok — so we want to avoid putting it online for now. The best way to do so is to move it into the drafts folder :)